cypress/e2e/pricing.cy.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+
+import { STRIPE_PRODUCT } from '../../pages/api/stripe/stripeProducts';
+
+//Check if the buttons works properly
+describe('Pricing landing', () => {
+  const DRIVE_WEB_URL = Cypress.env('DRIVE_WEB_URL');
+  const TWOTB_OFF_COUPON = 'P8PSpVs6';
+
+  const url = ({ planId, couponCode, mode }: { planId: string; couponCode?: string; mode?: string }) => {
+    return `${DRIVE_WEB_URL}/new?redirectUrl=${encodeURIComponent(
+      DRIVE_WEB_URL + '/checkout-plan',
+    )}${encodeURIComponent(
+      `?planId=${planId}${couponCode ? '&couponCode=' + couponCode : ''}&mode=${mode ? mode : 'subscription'}`,
+    )}`;
+  };
+
+  describe('Free plan button', () => {
+    describe('10GB free plan', () => {
+      it('should redirect to https://drive.internxt.com/new', () => {
+        cy.visit('/pricing', { timeout: 10000 });
+        cy.get('#priceTable').contains('Get Internxt for free').click();
+
+        cy.url().should('eq', 'https://drive.internxt.com/new');
+      });
+    });
+  });
+
+  describe('Paid plan button', () => {
+    describe('20GB plan', () => {
+      describe('Monthly', () => {
```

Comment:
If visually, the sections are in Monthly / Annually sections, I would join them using those categories instead of grouping by space, following a easy-to-find structure